src/Controller/NextEuropaDashboardController.php

```diff
@@ -205,6 +205,13 @@ function uli($ne_dashboard_agent_req_for_user = FALSE) {
    * @return \Drupal\Core\Access\AccessResultAllowed|\Drupal\Core\Access\AccessResultForbidden
    */
   public function access() {
+
+    $account = \Drupal::currentUser();
```

Comment:
`\Drupal` calls should be avoided in classes, please use dependency injection instead.

Reply:
Done